crates/bevy_render/src/camera/camera.rs

```diff
@@ -170,6 +170,10 @@ impl Camera {
     /// of the current [`RenderTarget`].
     ///  For logic that requires the full logical size of the
     /// [`RenderTarget`], prefer [`Camera::logical_target_size`].
+    ///
+    /// Returns None if both:
+    /// - The viewport is not set
+    /// - The render target is not set
```

Comment:
I think this might not be entirely correct.

To me, it looks like this would return `None` only if there is no render target, or if the projection matrix / render target have not yet been computed by `camera_system`.

Reply:
I agree with that, the "render target" cannot not be set, it's not an option, but the "computed render target",  `computed.target_info`, can not be set if the camera was just created or something.

Reply:
Ah, so I suppose it's just a matter of perspective. I am imagining reading this as a user with no knowledge of that internal implementation detail and it seems a bit confusing as-is from that angle.

As a user, I can set a custom viewport or not, and that doesn't really matter here.

As a user, the render target is not optional (but perhaps I can set it to something invalid?)

As a user, I might attempt to use these functions before `computed.whatever` is computed by `camera_system`.

I personally think that we should use this sort of user-facing language, or if we're talking about specific optional properties, maybe throw some backticks around them to make that more clear.

Reply:
I agree.

Actually, no matter if the viewport is set or not you need `Camera.computed.target_info`, so it doesn't matter at all if there is a viewport, and the user only needs to know that.

Something like "Returns `None` if the function is called just after the Camera is created, before `camera_system` is executed."

Reply:
Made a draft for this based on @Selene-Amanita's comment

Reply:
I investigated a bit more and edited my message, sorry @tormeh I saw that you already added my suggestion ^^'

Reply:
I think for `world_to_viewport` you can put a reference, like I think for `world_to_viewport` you can put a reference, like "The logical viewport size cannot be computed, see [logical_viewport_size](Camera::logical_viewport_size)"

Reply:
Done